addons/web_editor/static/src/xml/editor.xml

```diff
@@ -311,28 +311,28 @@
                 <img class="img img-fluid o_cropper_image" t-att-src="widget.imageData.originalSrc"/>
            </div>
            <div class="o_crop_options text-center mt16">
-               <div class="btn-group" title="Aspect Ratio" data-toggle="buttons">
+               <div class="btn-group btn-group-toggle" title="Aspect Ratio" data-toggle="buttons">
```

Comment:
The whole bug can be fixed by simply removing that strange `data-toggle="buttons"`. Where did you find documentation about the `btn-group-toggle` class ?

Reply:
@qsm-odoo after BS4 option buttons UI was break (I can't see which is active ratio?)
Previously we are using simple button group and toggle `active` class with JS code.
But now we use radio button group which automatically manage `active` class by BS(https://getbootstrap.com/docs/4.1/components/buttons/#checkbox-and-radio-buttons),
No need JS code for toggle `active` class anymore.

We remove `data-toggle="buttons"` even it's working bcz of our JS code (Which is removed in this commit).
On my side it working in firefox.

Reply:
Thanks, I could not find that in the doc before (I searched in btn-group but this is not documented there). Maybe I did not test correctly, I will check again.